jenkins/e2e-image/gcloud-common.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+# This script downloads, installs cloud sdk and activates the service account.
+# USAGE:
+#   gcloud-common.sh [--download] -t|--tar [gcloud-tarball-dir] -d|--dir [install-dir]
+#       --download - if need download gcloud tarball from web
+#       -t|--tar - where to find/fetch gcloud tarball
+#       -d|--dir - where to install gcloud to
+#
+
+
+# default
+DOWNLOAD=false
+TARBALL="${WORKSPACE}/google-cloud-sdk.tar.gz"
+INSTALL_DIR="/"
+
+while [[ $# -gt 1 ]]
```

Comment:
That works, although I want everything in e2e-runner.sh to move into hack/e2e.go in the near future.